services/m3aggregator/config/aggregator.go

```diff
@@ -46,6 +46,10 @@ import (
 	"github.com/spaolacci/murmur3"
 )
 
+const (
+	initialBufferSizeGrowthFactor = 2
```

Comment:
Hmm, in that case, perhaps we should just go with the simple approach of using 2x. I was thinking that the number of buffers would be closer to the number of connections (~15-20K) in which case the savings might be more significant, especially if we ever decided to increase the size of the buffers themselves.